config/environments/production.rb

```diff
@@ -136,7 +136,7 @@
     'X-Frame-Options' => 'ALLOWALL',
   }
 
-  config.assets.prefix = "#{ENV['RELATIVE_URL_ROOT'] ? '/' + ENV['RELATIVE_URL_ROOT'] : '/lti'}/assets"
+  config.assets.prefix = "#{ENV['RELATIVE_URL_ROOT'] ? "/#{ENV['RELATIVE_URL_ROOT']}" : '/lti'}/assets"
```

Comment:
Same as for production.rb.

One thing though. If the same setting is setup in both environments, we better move it to application.rb and remove it from development and production